torchtune/modules/tanh_gate.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+# All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+import torch
+
+from torch import nn, Tensor
+
+
+class TanhGate(nn.Module):
+    """Implements a basic learnable gate to scale layer outputs
+
+    Args:
+        dim (int): parameter dim
```

Comment:
which dimension is scaled? hidden dim? all dims?

Reply:
Should just be last dimension, right? But yeah would document that explicitly